uuid.go

```diff
@@ -161,6 +161,27 @@ func TimestampFromV7(u UUID) (Timestamp, error) {
 // zero.
 var Nil = UUID{}
 
+// Max is the maximum UUID, as specified in RFC-4122, that has all 128 bits
+// set to one.
+var Max = UUID{
```

Comment:
I think this should be called `Omni` to match the verbiage in the proposed update to the RFC.  This is not a "maximum" UUID value in the same way that `Nil` is not a "minimum" UUID.

The doc string should also include a callout that it's based on a proposed, but as yet unaccepted, revision and, as such, it is subject to change (or even be removed).

Reply:
Just checked the latest draft update [here](https://datatracker.ietf.org/doc/draft-ietf-uuidrev-rfc4122bis/) and this actually is called "Max UUID" now.  I still personally disagree with the name because it's not accurate, we should be consistent with the RFC.

Still need the doc string callout, though.

Reply:
I went ahead and tweaked it. Hope it's not a faux pas :)